opencog/haskell/CMakeLists.txt

```diff
@@ -18,11 +18,11 @@ INSTALL (TARGETS atomspace-cwrapper
 	DESTINATION "lib${LIB_DIR_SUFFIX}/opencog"
 )
 
-IF (HAVE_STACK)
-    ADD_CUSTOM_TARGET(haskell-atomspace-lib ALL
-        DEPENDS atomspace-cwrapper
-        COMMAND bash build.sh ${CMAKE_CURRENT_BINARY_DIR}
-        WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
-        COMMENT "Building Haskell bindings."
-    )
-ENDIF (HAVE_STACK)
+#IF (HAVE_STACK)
+#    ADD_CUSTOM_TARGET(haskell-atomspace-lib ALL
+#        DEPENDS atomspace-cwrapper
+#        COMMAND sh build.sh ${CMAKE_CURRENT_BINARY_DIR}
+#        WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
+#        COMMENT "Building Haskell bindings."
+#    )
+#ENDIF (HAVE_STACK)
```

Comment:
this has nothing at all with file loading .. if this is needed, please submit a separate pull req for it.